core/modules/replica/IngestServerConnection.cc

```diff
@@ -430,7 +430,13 @@ void IngestServerConnection::_loadDataIntoTable() {
         string const sqlDatabase = h.conn->sqlId(_databaseInfo.name);
         string const sqlPartition = h.conn->sqlPartitionId(_transactionId);
 
-        vector<string> statements;
+        vector<string> tableMgtStatements;
+
+        // Make sure no outstanding table locks exist from prior operations
+        // on persistent database connections.
+        tableMgtStatements.push_back("UNLOCK TABLES");
```

Comment:
Why do you need it, do you explicitly lock tables somewhere?

Reply:
I have a persistent connection pool for MySQL. This statements is to ensure no remaining locks are left on the connection from a previous client.